### PR TITLE
modem: ppp: Remove force PDN activation

### PR DESCRIPTION
**Pull request by icblnk**
Removed 1st PDN activation as it's up by default and it leads to the
modem to become unresponsive.

### CODE REVIEW

**Review by DunderRoffe — COMMENTED** [review]
Is this thoroughly tested?
AT+CGACT=1,1 was a hard requirement during board bring up to get a LTE connection.

**Comment by endian-benjamin** [comment]
I like replacing CGATT with CGACT... But I'd like to take another look at what the application notes suggests first.

Also I got the impression from Viktor that CGACT=1,1 was STILL required on his board - perhaps @DunderRoffe can try this PR out?

File: drivers/modem/gsm_ppp.c [hunk 1]
```diff
@@ -258,24 +260,44 @@ static struct setup_cmd setup_cmds2[] = {
...
-		error = 0;
-		LOG_INF("Attached to packet service!");
+	/* Expected response is "+CGREG: urc_enabled,status"
+	 * Status values:
+	 * 1 - registered, home network
+	 * 5 - registered, roaming
+	 */
+	if (argc > 1 && (atoi(argv[1]) == 1 || atoi(argv[1]) == 5)) {
+		gsm.gprs_registered = true;
+		LOG_INF("GPRS network registered.");
```
Comment (endian-benjamin) [inline drivers/modem/gsm_ppp.c]: Mise well print argv[1] too because it's sorta nice debug info

File: drivers/modem/gsm_ppp.c [hunk 2]
```diff
@@ -258,24 +260,44 @@ static struct setup_cmd setup_cmds2[] = {
...
+MODEM_CMD_DEFINE(on_cmd_atcmdinfo_eps_network_registration)
+{
+	/* Expected response is "+CEREG: urc_enabled,status"
+	 * Status values:
+	 * 1 - registered, home network
+	 * 5 - registered, roaming
+	 */
+	if (argc > 1 && (atoi(argv[1]) == 1 || atoi(argv[1]) == 5)) {
+		gsm.eps_registered = true;
+		LOG_INF("EPS network registered.");
```
Comment (endian-benjamin) [inline drivers/modem/gsm_ppp.c]: and here

**Comment by endian-benjamin** [comment]
It looks good but yeah Viktor needs to test this first...

File: drivers/modem/gsm_ppp.c (hunk 1 above)
Comment (icblnk) [reply drivers/modem/gsm_ppp.c]: done

File: drivers/modem/gsm_ppp.c (hunk 2 above)
Comment (icblnk) [reply drivers/modem/gsm_ppp.c]: done

**Comment by DunderRoffe** [comment]
Tested LGTM

**Review by endian-benjamin — APPROVED** [review]

**Pull request merged**